packaging/windows/WINDOWS_INSTALLER.md

```diff
@@ -48,3 +58,11 @@ netdata-installer.exe /S /A /TOKEN=<YOUR_TOKEN> /ROOMS=<YOUR_ROOM>
 ```
 
 Replace `<YOUR_TOKEN>` and `<YOUR_ROOM>` with your actual Netdata Cloud Space claim token and room ID, respectively.
+
+> **Note**
+>
+> If you are running the installation for the first time, MSYS2 will also initialize.
```

Comment:
> will also initialize

what does it mean?

Reply:
This was weird, maybe @thiagoftsm actually knows what happens in this case, a MSYS2 window popped up and I think it had nothing to do with Netdata, probably it was trying to initialize everything. After reinstalling, it didn't display the messages it did at first

Reply:
I mean, what is "MSYS2 initialization"?

Reply:
MSYS2 is a command line environment like Windows CMD when I fist ran the installer it started and produced an output that I didn't capture, but it looked like some sort of first-time run

Reply:
From the user's point of view, "MSYS2 is also initialized" is completely unclear. I suggest deleting this line or rewording.

Reply:
Given I don't know how it works, let's remove it and leave it to the user to assume that the installation invokes the MSYS2 window.